config/templates/cloud-config-worker

```diff
@@ -144,27 +144,32 @@ coreos:
     {{ if .Experimental.NodeDrainer.Enabled }}
     - name: kube-node-drainer.service
       enable: true
+      command: start
+      runtime: true
       content: |
         [Unit]
         Description=drain this k8s node to make running pods time to gracefully shut down before stopping kubelet
-        Wants=kubelet.service docker.service
+        After=multi-user.target
 
         [Service]
         Type=oneshot
         RemainAfterExit=true
-        ExecStartPre=/usr/bin/systemctl is-active kubelet.service
-        ExecStartPre=/usr/bin/systemctl is-active docker.service
         ExecStart=/bin/true
-        ExecStop=/bin/sh -c '/usr/bin/docker run --rm -v /etc/kubernetes:/etc/kubernetes {{.HyperkubeImageRepo}}:{{.K8sVer}} \
-          /hyperkube kubectl \
+        TimeoutStopSec=30s
```

Comment:
Got it!

I was just reading https://www.freedesktop.org/software/systemd/man/systemd.service.html#TimeoutStopSec= which says that `TimeoutStopSec` defaults to the system-wide value configured via `DefaultTimeoutStopSec`.

Regardless of the default, setting enough timeout like this would be good.
This is what we couldn't have without your effort 👍